deploy/charts/crate-operator/templates/rbac.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if .Values.rbac.create -}}
```

Comment:
Do we need to have an `if` for the rbac? When would we not want to install this?

Reply:
As the ClusterRole and ClusterRoleBinding are not linked to a specific namespace, I would rather let the user the possibility to disable it.
I don't think it should happen often, or maybe never, but I'd like better to have the possibility in case it happens instead of not being able to continue the installation.